openhands/datasets/isolated/bhosporus22k.py

```diff
@@ -0,0 +1,46 @@
+import os
+import pandas as pd
+from glob import glob
+from .base import BaseIsolatedDataset
+from ..data_readers import load_frames_from_video
+
+class Bhosporus22kDataset(BaseIsolatedDataset):
+    """
+    Turkis Isolated Sign language dataset(Bhosporus22k) from the paper:
+    Link to paper: https://arxiv.org/pdf/2004.01283.pdf
+    """
+    def read_glosses(self):
+        df = pd.read_csv(self.class_mappings_file_path, delimiter=",")
+        print(self.class_mappings_file_path)
+        all_glosses = [df.iloc[i][2] for i in range(len(df))]
+        unique_glosses=set(all_glosses)
+        self.glosses = list(unique_glosses)
+        #print(self.glosses)
```

Comment:
Please remove lines like this after debugging/development is done.

Reply:
Will remove those lines and will update the file.